staging/utils/runtimex/runtime.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/seal-io/walrus/utils/log"
 )
 
-func init() {
+func Init() {
```

Comment:
It is implicitly invoked before. Any impact of this change? Why do we need the change in runner.go?

Reply:
This implicitly invoked will always print the log inside the Set function while using log package, like below:
`maxprocs        maxprocs: Leaving GOMAXPROCS=%s: CPU quota undefined`

Reply:
Can we change Printf to Debugf?